error: '::clock' has not been declared

when trying to compile miam-player on Fedora27 with gcc7 the following error message appears:

```
g++ -c -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -m64 -mtune=generic -g -std=gnu++11 -Wall -W -D_REENTRANT -fPIC -DUSE_SWRESAMPLE=1 -DMIAMACOUSTID_LIBRARY -DQT_MULTIMEDIA_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_SQL_LIB -DQT_CORE_LIB -I. -Ichromaprint -Ichromaprint/libavutil -Ichromaprint -I../core -I../core/3rdparty -I../core/3rdparty/QtAV -isystem /usr/include/qt5 -isystem /usr/include/qt5/QtMultimedia -isystem /usr/include/qt5/QtWidgets -isystem /usr/include/qt5/QtGui -isystem /usr/include/qt5/QtNetwork -isystem /usr/include/qt5/QtSql -isystem /usr/include/qt5/QtCore -Idebug/.moc -isystem /usr/include/libdrm -I. -I/usr/lib64/qt5/mkspecs/linux-g++ -o debug/.obj/mbrelease.o mbrelease.cpp
make[1]: Leaving directory '/builddir/build/BUILD/Miam-Player-1a21b01a86c4cbcfe8fc6d99cf6e595838856b11/src/acoustid'
In file included from /usr/include/c++/7/chrono:41:0,
                 from /usr/include/qt5/QtCore/qobject.h:59,
                 from /usr/include/qt5/QtCore/QObject:1,
                 from mbrelease.h:5,
                 from mbrelease.cpp:1:
/usr/include/c++/7/ctime:64:11: error: '::clock' has not been declared
   using ::clock;
           ^~~~~
/usr/include/c++/7/ctime:65:11: error: '::difftime' has not been declared
   using ::difftime;
           ^~~~~~~~
/usr/include/c++/7/ctime:66:11: error: '::mktime' has not been declared
   using ::mktime;
           ^~~~~~
/usr/include/c++/7/ctime:67:11: error: '::time' has not been declared
   using ::time;
           ^~~~
/usr/include/c++/7/ctime:68:11: error: '::asctime' has not been declared
   using ::asctime;
           ^~~~~~~
/usr/include/c++/7/ctime:69:11: error: '::ctime' has not been declared
   using ::ctime;
           ^~~~~
/usr/include/c++/7/ctime:70:11: error: '::gmtime' has not been declared
   using ::gmtime;
           ^~~~~~
/usr/include/c++/7/ctime:71:11: error: '::localtime' has not been declared
   using ::localtime;
           ^~~~~~~~~
/usr/include/c++/7/ctime:72:11: error: '::strftime' has not been declared
   using ::strftime;
           ^~~~~~~~
make[1]: *** [Makefile:555: debug/.obj/mbrelease.o] Error 1
make[1]: *** Waiting for unfinished jobs....
```
full error message http://koji.rpmfusion.org/kojifiles/work/tasks/878/160878/build.log

--- a/src/acoustid/acoustid.pro
+++ b/src/acoustid/acoustid.pro
@@ -1,6 +1,10 @@
 QT          += core gui multimedia network sql widgets
 TEMPLATE    = lib
-include(chromaprint/chromaprint.pri)
+
+# Use system headers on Unix
+win32 {
+    include(chromaprint/chromaprint.pri)
+}
 
 DEFINES += MIAMACOUSTID_LIBRARY
 

--- a/src/acoustid/qchromaprint.h
+++ b/src/acoustid/qchromaprint.h
@@ -1,7 +1,7 @@
 #ifndef QCHROMAPRINT_H
 #define QCHROMAPRINT_H
 
-#include <chromaprint/chromaprint.h>
+#include <chromaprint.h>
 
 #include <QObject>
 

